src/Mono.Android/Mono.Android.targets

```diff
@@ -128,6 +129,7 @@
         SourceFiles="$(IntermediateOutputPath)__dex\classes.dex"
         DestinationFiles="$(OutputPath)mono.android.dex"
     />
+    <Touch Files="$(OutputPath)mono.android.dex" />
```

Comment:
Alright. We should completely eliminate those use of "AfterTargets", but that can be done in different commits.